.github/dependabot.yml

```diff
@@ -48,8 +34,11 @@ updates:
       # Addon framework, this shouldn't be upgraded on release branches
       - dependency-name: open-cluster-management.io/addon-framework
       - dependency-name: open-cluster-management.io/api
+      # These are included with submariner-operator
+      - dependency-name: github.com/submariner-io/admiral
+      - dependency-name: github.com/submariner-io/submariner
       # 2.6 tracks Submariner 0.13
-      - dependency-name: github.com/submariner-io/*
+      - dependency-name: github.com/submariner-io/submariner-operator
```

Comment:
Don’t we need to track `cloud-prepare` too?

Reply:
yeah